use_case_examples/cifar/cifar_brevitas_training/README.md

```diff
@@ -98,7 +98,7 @@ Experiments were conducted on an m6i.metal machine offering 128 CPU cores and 51
 | VGG FHE (simulation\*) | 6 bits   | 86.0     |
 | VGG FHE                | 6 bits   | 86.0\*\* |
 
-We ran the FHE inference over 10 examples and achieved 100% similar predictions between the simulation and FHE. The overall accuracy for the entire data-set is expected to match the simulation. The original model with a maximum of 13 bits of precision ran in around 9 hours on the specified hardware. Using the rounding approach, the final model ran in **31 minutes**, providing a speedup factor of 18x while preserving accuracy. This significant performance improvement demonstrates the benefits of the rounding operator in the FHE setting.
+We ran the FHE inference over 10 examples and achieved 100% similar predictions between the simulation and FHE. The overall accuracy for the entire data-set is expected to match the simulation. The original model (no rounding) with a maximum of 13 bits of precision runs in around 9 hours on the specified hardware. Using the rounding approach, the final model ran in **4 minutes**. This significant performance improvement demonstrates the benefits of the rounding operator in the FHE setting.
```

Comment:
I think it's worth tracking the performance improvement. Something like: jan 2023: 31min, august 2023: 4min.

Reply:
Not sure if useful for the user

Reply:
they are supposed to be up to date for the past 3 months at most, so I also don't think it's very relevant !

Reply:
I would be reassured to see that there was a 30x speed up in the past 6 months. But I heard the masses. Let's not do it.